includes/razorpay-webhook.php

```diff
@@ -54,28 +54,45 @@ public function process()
                     write_log(['message' => $e->getMessage()]);
                     return;
                 }
-            }
 
-            switch ($data['event'])
-            {
-                case 'payment.authorized':
-                    return $this->paymentAuthorized($data);
+                switch ($data['event'])
```

Comment:
I meant, we missed this in the last release. Which means the present release has a security issue.